src/server/src/main/java/io/cassandrareaper/ReaperApplication.java

```diff
@@ -427,6 +430,19 @@ private void checkConfiguration(ReaperApplicationConfiguration config) {
     LOG.debug("repairParallelism: {}", config.getRepairParallelism());
     LOG.debug("hangingRepairTimeoutMins: {}", config.getHangingRepairTimeoutMins());
     LOG.debug("jmxPorts: {}", config.getJmxPorts());
+
+    if (config.getHttpManagement() != null) {
+      if (config.getHttpManagement().isEnabled()) {
+        if (config.getHttpManagement().getTruststoresDir() != null) {
+          if (!Files.exists(Paths.get(config.getHttpManagement().getTruststoresDir()))) {
+            throw new RuntimeException(String.format(
```

Comment:
issue: There's odd logic here. If getTrustStoresDir() is null, then we won't have any RuntimeException.

Reply:
Or do you rely on the truststore value to cause the RuntimeException somewhere else?

Reply:
Yeah I've added it here, in a `checkConfiguration()`, so that it fails early. Then the stacktrace is smaller and more obvious.

Otherwise, the RuntimeException would happen later, when it's trying to setup the watchers for changes, which in turn happens when actually adding a cluster. The stack traces at this point are incredibly long and nested.